src/js/components/user/user.js

```diff
@@ -45,8 +45,8 @@ class User extends MaharaBaseComponent {
             icon = defaultIcon;
             displayName = "";
         }
-
-
+        // add timestamp to query to prevent image caching, because we're using same name 
+        icon += `?${Date.now() || ''}`;
```

Comment:
is there anything else we can use that isn't so specific? this means the image will never be cached - if the version string is available that could be better?